packages/camera/camera/ios/Classes/CameraPlugin.m

```diff
@@ -371,6 +370,8 @@ - (instancetype)initWithCameraName:(NSString *)cameraName
   _focusMode = FocusModeAuto;
   _lockedCaptureOrientation = UIDeviceOrientationUnknown;
   _deviceOrientation = orientation;
+  // Format used for video and image streaming.
```

Comment:
Comments about the purpose of a variable should be on the declaration (in this case, the property) not the assignment.

Reply:
I fixed it.